_posts/en/newsletters/2019-10-09-newsletter.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 'Bitcoin Optech Newsletter #67'
+permalink: /en/newsletters/2019/10/09/
+name: 2019-10-09-newsletter
+slug: 2019-10-09-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests help testing release candidates for
+Bitcoin Core and LND, tracks continued discussion about the proposed
+noinput and anyprevout sighash flags, and describes several notable
+changes to popular Bitcoin infrastructure projects.
+
+{% comment %}<!-- include references.md below the fold but above any Jekyll/Liquid variables-->{% endcomment %}
+{% include references.md %}
+
+## Action items
+
+- **Help test Bitcoin Core 0.19.0RC1:** production users of Bitcoin Core
+  are especially encouraged to test this [latest release candidate][core
+  0.19.0] to ensure that it fulfills all of your organization's needs.
+  Experienced users who plan to test are also asked to take a few
+  moments to test the GUI and look for problems that might
```

Comment:
This is something that we've asked before (e.g. https://bitcoinops.org/en/newsletters/2018/12/11/#help-test-bitcoin-core-0-17-1rc1 ).  I can't quickly find a reference, but it stems from a an issue in the, I think, 0.17.0 release where there was a GUI issue that was missed for a long time during the RC cycle, possibly because we only encourage advanced users and sysadmins to test and many of them don't use the GUI.  There's no special motivation for GUI testing this release, just the generic motivation explained in the current text.